submissions/krylenger/a-tiny-JS-world_post_OOP/index.js

```diff
@@ -0,0 +1,140 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details */
+
+const dog = {
+    type: "animal",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    friends: ["Anna", "Eugene"],
```

Comment:
So, all dogs in this world will all be males and will all have Anna and Eugene as their friends?
Doesn't look really realistic. 
Can we allow more life-like customization?
There is no much point of defining class templates to create just one instance of each inhabitant class.

Please apply the same to all classes.

Reply:
✅ Done